model/fv_arrays.F90

```diff
@@ -748,6 +748,8 @@ module fv_arrays_mod
                                           !< is recommended to only set this to .true. when initializing the model.
    logical :: fv_debug  = .false.  !< Whether to turn on additional diagnostics in fv_dynamics.
                                    !< The default is .false.
+   logical :: fv_timers  = .false. !< Whether to turn on performance metering timers in the dycore and moving nest
```

Comment:
This is a nice addition; it could be extended in the future to add more fine-grained timers.

Reply:
Actually, the granularity of output for MPP clocks is determined by an FMS namelist variable that works with the various timer levels specified in the clock_id to determine at what level a clock will be output.